tests/conftest.py

```diff
@@ -7,7 +7,11 @@
 from twister2.platform_specification import PlatformSpecification
 from twister2.twister_config import TwisterConfig
 
-pytest_plugins = ['pytester']
+pytest_plugins = [
+    'pytester',
+    'twister2.report.subtest_reporting_plugin',
```

Comment:
I don't think those plugins should be registered here as they already registered by twister plugin.

Reply:
It was an attempt to resolve CI failures, which I cannot see locally. I see it makes no sense, will remove it

Reply:
fixed